sequel/lib/criteriaProcessor.js

```diff
@@ -691,6 +691,11 @@ CriteriaProcessor.prototype.prepareCriterion = function prepareCriterion(key, va
       }
 
       break;
+
+    default:
+      var err = new Error('Unknown filtering operator: "' + key + "\". Should be 'startsWith', '>', 'contains' or similar");
+      err.operator = key;
+      throw err;
```

Comment:
Yeah, I don't think I said it very well!  Here's a good resource to get an idea of the issue and the workaround: https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#2-unsupported-syntax